src/reactome/metadata_info.py

```diff
@@ -31,7 +29,6 @@
             description=pathway_name_description,
             type="string",
         ),
-        AttributeInfo(name="summation", description="The descriptions of the pathway"),
```

Comment:
This appears to be new and I'm not sure if it's supposed to be here. It's missing the `type` param.